ZeekPluginDynamic.cmake

```diff
@@ -95,7 +95,12 @@ if ( NOT ZEEK_PLUGIN_INTERNAL_BUILD )
                             ${CMAKE_CURRENT_SOURCE_DIR}/src
                             )
 
-        set(ENV{PATH} "${BRO_PLUGIN_BRO_BUILD}/build/src:$ENV{PATH}")
+        if ( WIN32 )
```

Comment:
Most of the other places there's a check for Windows, it's checking for `if ( MSVC )`. Should this be the same?